provisioners/windows/geckotwin1064ht-bootstrap.ps1

```diff
@@ -48,9 +48,10 @@ function Set-RoninRegOptions {
     [string] $ronnin_key = "$mozilla_key\ronin_puppet",
     [string] $source_key = "$ronnin_key\source",
     [string] $workerType = 'gecko-t-win10-64-ht',
-    [string] $src_Organisation = 'mozilla-platform-ops',
```

Comment:
I assume these changes pointing to your repo are temporary.  When this goes into production for windows in the data center we shouldn't do this even temporarily.